### PR TITLE
Add version infos about rom

### PR DESCRIPTION
**Pull request by KikiManjaro**
I have added version of rom infos inside:

<details> 
  <summary>App dialog</summary>
  
Screenshot_2022-08-07-03-45-57-286_skyline emu

</details>

<details> 
  <summary>App Item Grid</summary>

Screenshot_2022-08-07-03-45-51-631_skyline emu

</details>

<details> 
  <summary>App Item Grid Compact</summary>
  
Screenshot_2022-08-07-03-46-11-331_skyline emu

</details>

<details> 
  <summary>App Item List</summary>

Screenshot_2022-08-07-03-45-38-900_skyline emu

</details>

I'm not really sure if it is usefull to add it for app items, maybe adding it to the dialog could be good enough feel free to tell me :)

Note: This is the first time ever i've touched to cpp files, i've partially understand how it was working and made change to make it work, feel free to tell me if i made any mistake

Note 2: I've seen that there is a lot of informations that could be helpfull to users like `Time player`, `Last Time Played`, `File Size` and other that i'd like to add if you think there is an interest

### CODE REVIEW

**Comment by Tiredasfunk** [comment]
bro you're doing a good job but stop spamming in the action tab

**Comment by Sovietfowl** [comment]
They are not spamming anything, this is how pull requests work.

**Comment by Tiredasfunk** [comment]
> They are not spamming anything, this is how pull requests work.

so what happens when the workflow gets approved?

**Comment by Sovietfowl** [comment]
It will get approved when all of the devs agreed it's ready to be merged.

**Comment by Tiredasfunk** [comment]
> It will get approved when all of the devs agree it's ready to be merged.

ok thanks for helping me understand the process.

**Comment by Joko1994** [comment]
Add dock menu in game play .. some game need that

**Review by bylaws — APPROVED** [review]
Lgtm, this is very useful!

**Comment by bylaws** [comment]
You're doing fine with CPP btw, looks good

**Comment by KikiManjaro** [comment]
> bro you're doing a great job but please stop spamming in the action tab

Did i spam ? I'm sorry if that's the case, i can try to push less often, I know I have a bad habit of commit and directly pushing. I'll do my best :)

**Comment by Sovietfowl** [comment]
It's fine don't worry about them Kilimanjaro, they are just unaware how GitHub works.

File: app/src/main/java/emu/skyline/data/DataItem.kt [hunk 1]
```diff
@@ -39,6 +39,11 @@ data class AppItem(private val meta : AppEntry) : DataItem() {
      */
     val uri get() = meta.uri
 
+    /**
+     * The string used as the sub-title, we currently use the author
+     */
```
Comment (lynxnb) [inline app/src/main/java/emu/skyline/data/DataItem.kt:44]: `The application version` or something like that.

**Review by lynxnb — CHANGES_REQUESTED** [review]
LGTM too, just a small thing about a comment.

File: app/src/main/java/emu/skyline/data/DataItem.kt [hunk 2]
```diff
@@ -39,6 +39,11 @@ data class AppItem(private val meta : AppEntry) : DataItem() {
      */
     val uri get() = meta.uri
 
+    /**
+     * The string used as the sub-title, we currently use the author
```
Comment (KikiManjaro) [inline app/src/main/java/emu/skyline/data/DataItem.kt:43]:
```suggestion
     * The string used as the application version, we currently use the author
```

File: app/src/main/java/emu/skyline/data/DataItem.kt (hunk 1 above)
Comment (KikiManjaro) [reply app/src/main/java/emu/skyline/data/DataItem.kt]: Oh yeah, i think i forget to change it after a copy and paste :) thks

File: app/src/main/java/emu/skyline/data/DataItem.kt (hunk 2 above)
Comment (lynxnb) [reply app/src/main/java/emu/skyline/data/DataItem.kt]: Drop the part after the comma and you're good.

**Comment by AbandonedCart** [comment]
Seems a bit excessive beyond the dialog and list if it isn't separately optional from displaying any info at all. In the list, it helps fill in the whitespace and in the dialog, it's practically what is expected. In compat grid, it begins to overpower the actual icon. In the regular grid, it wouldn't make sense if it wasn't also in the compat.

File: app/src/main/java/emu/skyline/data/DataItem.kt [hunk 3]
```diff
@@ -39,6 +39,11 @@ data class AppItem(private val meta : AppEntry) : DataItem() {
      */
     val uri get() = meta.uri
 
+    /**
+     * The string used as the application version, we currently use the author
```
Comment (lynxnb) [inline app/src/main/java/emu/skyline/data/DataItem.kt:43]:
```suggestion
     * The application version
```

**Review by lynxnb — CHANGES_REQUESTED** [review]

**Review by lynxnb — APPROVED** [review]
LGTM! We'll merge after you squash the commits. Also thanks for the coauthor, but no need to, it's your code after all ;)

**Comment by KikiManjaro** [comment]
> LGTM! We'll merge after you squash the commits. Also thanks for the coauthor, but no need to, it's your code after all ;)

It makes you the coautor automatically when a apply one of your suggestion :) you helpped me a lot that's normal

**Pull request merged**